inventory/api/compute/v1/compute.proto

```diff
@@ -223,6 +223,7 @@ message HostResource {
   ]; // textual message that describes the AMT status of Host. Set by DM RM only.
   status.v1.StatusIndication amt_status_indicator = 95 [(ent.field) = {optional: true}]; // Indicates dynamicity of the amt_status. Set by DM and OM RM only.
   uint64 amt_status_timestamp = 96 [(ent.field) = {optional: true}]; // UTC timestamp when amt_status was last changed. Set by DM and OM RM only.
+  uint32 lvm_size = 97 [(ent.field) = {optional: true}]; // LVM size in GB.
```

Comment:
Where do we set the default `lvm_size` to 0? We need to set the default size as 0, if user pass value while registering the device then it should be updated.